tests/Helper/HtmlFormTest.php

```diff
@@ -81,4 +96,21 @@ public function testGetInputNameException(): void
         $this->expectExceptionMessage('formName() cannot be empty for tabular inputs.');
         HtmlForm::getInputName($anonymousForm, '[0]dates[0]');
     }
+
+    public function testUUIDInputName(): void
+    {
+        $fields = $this->dynamicFieldsProvider('DynamicFieldsForm');
```

Comment:
I suggest to use the classic way for working with data providers here. https://phpunit.de/manual/3.7/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.data-providers